packages/db/drizzle/0056_backfill_balances_competition_id.sql

```diff
@@ -0,0 +1,44 @@
+-- Phase 2: Backfill competition_id for existing balances
```

Comment:
This is a new approach to backfilling than we're used to - Drizzle accepts plain SQL migrations and this migration file will backfill during the migration process.